packages/broker/src/plugins/testnetMiner/config.schema.json

```diff
@@ -5,9 +5,11 @@
     "description": "Testnet miner plugin configuration",
     "additionalProperties": false,
     "properties": {
-        "rewardStreamId": {
-            "type": "string",
-            "default": "streamr.eth/testnets/brubeck-rewards"
+        "rewardStreamIds": {
+            "type": "array",
+            "items": { "type": "string" },
+            "uniqueItems": true,
+            "default": ["streamr.eth/testnets/brubeck-rewards"]
```

Comment:
The default value should contain list of all testnet reward streams? @juslesan Do we know the stream IDs already?

Reply:
No, the real names for the reward streams are not known yet. There's a need to make a configuration PR once all urgent  changes are completed